### DISCUSSION TITLE
An unexpected internal error occurred when create git repository

### DISCUSSION THREAD

**Discussion by ZYSzys** [vercel/community#3251] [Help] [answered]
### Summary

An unexpected internal error occurred when create git repository

### Steps to Reproduce

Deploy from https://vercel.com/templates/next.js/app-directory
<img width="1722" alt="Screenshot 2023-07-15 at 01 55 13" src="https://github.com/vercel/community/assets/23313266/b706d263-e51e-424c-a843-3f24e5903749">

**Comment by amyegan** [comment]
Hey @ZYSzys. I wasn't able to replicate this error. There may have been a temporary communication error between Vercel and GitHub while creating the repo. Do you get the same error if you go back and try again?

**Reply by ZYSzys** [reply]
Yes, I can reproduce it stably under my account, and I found the  `/api/v1/integrations/git-repo` API return status code 500 

![image](https://github.com/vercel/community/assets/23313266/2c448e7c-7d7e-4325-b0c2-a1bfd330a6f8)

**Reply by owengot** [reply]
Have the same error.

**Reply by amyegan** [reply, marked as answer]
Sometimes login connections can be corrupted and cause import issues. Please try revoking GitHub login access to your Vercel account and then add it back again in your [account login connection settings](https://vercel.com/account/login-connections) 🙇

**Reply by tamlyn** [reply]
For me the problem was that I needed to grant the Vercel GitHub app some extra permissions here: https://github.com/settings/installations

**Comment by tmmsmoreira** [comment]
I am trying to create a new project, after revoking GitHub login access and adding it back again, the error stays the same "An unexpected internal error occurred". It is my first time using Vercel.
